meeting_minutes/2024/2024-12-19.md

```diff
@@ -0,0 +1,124 @@
+![image](https://user-images.githubusercontent.com/1690898/139102180-5c1e2583-14f1-4f58-ab2b-9e3807ed529c.png)
+
+# Common Security Advisory Framework (CSAF) Technical Committee Working Meeting
+
+- Meeting Date: December 19, 2024
+- Time: 17:00 UTC (19:00 CEST, 13:00 EDT, 10:00 PDT)
+
+## Call to Order and Welcome
+
+Meeting called to order @ 17:06 UTC
+
+## Roll call
+
+Inability to register attendees due to OASIS system challenges.
+
+## Participants
+
+| Given Name | Family Name | Affiliation                                                 | Role          |
+|:-----------|:------------|:------------------------------------------------------------|:--------------|
+| Christian  | Banse       | Fraunhofer-Gesellschaft e.V. - Fraunhofer AISEC             | Voting Member |
+| Christoph  | Plutte      | Ericsson AB                                                 | Voting Member |
+| Denny      | Page        | Individual                                                  | Voting Member |
+| Dina       | Truxius     | Federal Office for Information Security (BSI)               | Voting Member |
+| Feng       | Cao         | Oracle                                                      | Voting Member |
+| JD         | Stefaniak   | Dell Technologies                                           | Member        |
+| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Voting Member |
+| Martin     | Prpic       | Red Hat                                                     | Voting Member |
+| Omar       | Santos      | Cisco Systems                                               | Chair         |
+| Sonny      | van Lingen  | Huawei Technologies Co., Ltd.                               | Voting Member |
+| Stefan     | Hagen       | Individual                                                  | Voting Member |
+| Thomas     | Schaffer    | Cisco Systems                                               | Voting Member |
```

Comment:
Same here. I don't have Thomas on the list used for quorum. Was he in the recording?

Reply:
Yes.